CHANGELOG.md

```diff
@@ -63,10 +83,27 @@ This section is for maintaining a changelog for all breaking changes for the cli
 ### Fixed
 - Fix partial success results for msearch_template ([#709](https://github.com/opensearch-project/opensearch-java/pull/709))
 - Fix deserialization of node stats response ([#745](https://github.com/opensearch-project/opensearch-java/pull/745))
+- Fix PutIndexTemplateRequest field deserialization ([#765](https://github.com/opensearch-project/opensearch-java/pull/765))
 
```

Comment:
Line 37 and Line 38 also got released in 2.8.2. They might have been incorrectly added to Unreleased 3.0 section. Can you please add them here?